dbee/conn/history.go

```diff
@@ -393,4 +394,13 @@ func (r *HistoryRows) Next() (models.Row, error) {
 }
 
 func (r *HistoryRows) Close() {
+	// no-op
+}
+
+func (r *HistoryRows) SetCallback(callback func()) {
+	// no-op
+}
+
+func (r *HistoryRows) SetCustomHeader(header models.Header) {
+	// no-op
```

Comment:
Are these methods an artifact of the deleted interface?

Reply:
Indeed they are [c82b85e](https://github.com/kndndrj/nvim-dbee/pull/34/commits/c82b85e230b61eec0664f2e0e74e23decc516d29)